python/pyspark/sql/functions.py

```diff
@@ -4697,6 +4714,13 @@ def array_repeat(col: "ColumnOrName", count: Union["ColumnOrName", int]) -> Colu
 
     .. versionadded:: 2.4.0
 
+    Parameters
+    ----------
+    col : :class:`~pyspark.sql.Column` or str
+        column name or column that contains the element to be repeated
```

Comment:
nit: `column name or column containing the element to be repeated` to get consistency with other descriptions ?